test/functional/pdo_sqlsrv/pdo_testDataType.phpt

```diff
@@ -1,494 +0,0 @@
---TEST--
-Test the different type of data for retrieving
```

Comment:
This test was deleted because it is a duplication of pdostatement_GetDataType.phpt